src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -6,15 +6,14 @@ import {CommonActions, useNavigation} from '@react-navigation/native';
 import {atom, useRecoilState} from 'recoil';
 
 import AnonymousMessageRepo from '../../../service/repo/anonymousMessageRepo';
+import ChannelList from '../../../database/schema/ChannelListSchema';
 import SignedMessageRepo from '../../../service/repo/signedMessageRepo';
 import UseChatUtilsHook from '../../../../types/hooks/screens/useChatUtilsHook.types';
 import UserSchema from '../../../database/schema/UserSchema';
 import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
+import useUserAuthHook from '../auth/useUserAuthHook';
 import {ANON_PM, GROUP_INFO} from '../constant';
-import {
-  BetterSocialChannelType,
-  ChannelList
-} from '../../../../types/database/schema/ChannelList.types';
+import {BetterSocialChannelType} from '../../../../types/database/schema/ChannelList.types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ChannelList` has been moved from a grouped import statement to a separate line. This change doesn't affect the functionality, but it's important to maintain consistency in how you import modules across your codebase. If you're using separate lines for each import, stick with that. If you're grouping related imports together, continue doing so.

```diff
- import {
-   BetterSocialChannelType,
-   ChannelList
- } from '../../../../types/database/schema/ChannelList.types';
+ import ChannelList from '../../../database/schema/ChannelListSchema';
+ import {BetterSocialChannelType} from '../../../../types/database/schema/ChannelList.types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->